csharp/extractor/Semmle.Extraction.CSharp.DependencyFetching/DependabotProxy.cs

```diff
@@ -0,0 +1,77 @@
+using System;
+using System.Diagnostics;
+using System.IO;
+using Semmle.Util;
+using Semmle.Util.Logging;
+
+namespace Semmle.Extraction.CSharp.DependencyFetching
+{
+    public class DependabotProxy
```

Comment:
Maybe, but I am not sure I see any immediate benefit to it. Although I guess it would simplify the implementation for the feed reachability checking? Happy to do that in a follow-up PR if it makes sense, but it would be nice to get this functionality shipped for now.